docker/development.yml

```diff
@@ -125,7 +125,7 @@ services:
     environment:
       # In development and testing, the SSO service needs to contact the Supabase
       # service directly via Docker vs through the http://localhost/v1/supabase domain.
-      - SUPABASE_URL=http://kong:8000
+      - SUPABASE_URL=${SUPABASE_URL}
```

Comment:
Make this:

```yml
- SUPABASE_URL=${SUPABASE_URL:-http://kong:8000}
```

That way you can override it, but it defaults to what we want.

Reply:
What does `:-` just before **http** mean?   Can we only specify just `:`?

Reply:
Good question. See https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution:

> Both $VARIABLE and ${VARIABLE} syntax are supported. Additionally when using the [2.1 file format](https://docs.docker.com/compose/compose-file/compose-versioning/#version-21), it is possible to provide inline default values using typical shell syntax:
>
>${VARIABLE:-default} evaluates to default if VARIABLE is unset or empty in the environment.
>${VARIABLE-default} evaluates to default only if VARIABLE is unset in the environment.
>Similarly, the following syntax allows you to specify mandatory variables:
>
>${VARIABLE:?err} exits with an error message containing err if VARIABLE is unset or empty in the environment.
>${VARIABLE?err} exits with an error message containing err if VARIABLE is unset in the environment.

Reply:
Thank you, fixed.